src/main/java/com/gooddata/project/ProjectValidation.java

```diff
@@ -0,0 +1,47 @@
+package com.gooddata.project;
+
+import org.codehaus.jackson.annotate.JsonCreator;
+import org.codehaus.jackson.annotate.JsonValue;
+
+/**
+ * Represents one project validation.
+ */
+public class ProjectValidation {
```

Comment:
I'd vote for different name like ProjectValidationOption ... and the javadoc is also misleading